src/cinder/Text.cpp

```diff
@@ -675,21 +675,28 @@ vec2 TextBox::measure() const
 Surface	TextBox::render( vec2 offset )
 {
 	createLines();
-	
+
 	float sizeX = ( mSize.x <= 0 ) ? mCalculatedSize.x : mSize.x;
-	float sizeY = ( mSize.y <= 0 ) ? mCalculatedSize.y : mSize.y;
+	float sizeY;
+	if ( mSize.y <= 0 ) {
+		sizeY = (mCalculatedSize.y + (mLeading * mLines.size()));
+	} else {
+		sizeY = (mSize.y + (mLeading * mLines.size()));
```

Comment:
To sum up the above; I think we double-check the math on the leading calculation, and incorporate the leading calculation into the calculated size member variable rather that tack in on when rendering.